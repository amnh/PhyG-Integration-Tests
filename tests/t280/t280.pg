--build(graph:hardwired), inputting fastc files along with tcm
set(seed:1634561640)
set(graphtype:hardwired)
read(rename:"woman_rename.txt")
read(fastc:"woman.fastc", tcm: "woman-11.tcm")
read(fastc:"woman_cox1_spaced.fastc")
read(include:"Include_woman16.txt")
build(distance, rdwag, block, cun, displaytrees:10, atrandom, graph)
report("t280.csv", data, overwrite)
report("t280.tre", newick, overwrite)
report("t280.dot", dotpdf, overwrite)