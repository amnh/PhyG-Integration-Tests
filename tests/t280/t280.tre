(AlbanianTosk:24.5,(Danish:36.0,(((OldPrussian:36.0,(Faroese:26.0,Icelandic:58.0)HTU25:18.5)HTU23:12.5,((((Cornish:27.5,Manx:22.5)HTU18:25.5,#HTU32:293.5)HTU27:17.5,(Yiddish:31.5,(EasternFrisianSaterland:33.5,#HTU33:294.5)HTU28:15.5)HTU30:19.0)HTU26:18.5,(Assamese:42.0,Scots:40.0)HTU29:19.5)HTU24:19.0)HTU22:8.0,(ArmenianWestern:31.0,((SanskritVedic:33.5,(AncientIonicGreek:40.5,(Irish:293.5)#HTU32:293.5)HTU19:23.0)HTU20:21.5,(KolschColognian:298.5)#HTU33:298.5)HTU17:19.0)HTU34:17.5)HTU21:15.0)HTU31:27.5)HTU16[878.0];
(AlbanianTosk:24.5,(Danish:35.0,(((OldPrussian:34.0,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:13.5,(((Cornish:29.5,Manx:20.5)HTU17:25.0,(Yiddish:33.0,(EasternFrisianSaterland:34.0,KolschColognian:47.0)HTU26:14.5)HTU28:22.0)HTU25:17.5,(Assamese:42.0,Scots:40.0)HTU27:20.0)HTU23:17.5)HTU21:10.5,(ArmenianWestern:26.5,(SanskritVedic:31.0,(AncientIonicGreek:41.5,Irish:32.5)HTU18:31.0)HTU19:26.0)HTU30:21.0)HTU20:16.0)HTU29:29.0)HTU16[775.0];
(AlbanianTosk:24.5,(Danish:35.0,(((OldPrussian:34.0,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:13.5,(((Cornish:29.5,Manx:20.5)HTU17:25.0,(Yiddish:33.0,(EasternFrisianSaterland:34.0,KolschColognian:47.0)HTU26:14.5)HTU28:22.0)HTU25:17.5,(Assamese:42.0,Scots:40.0)HTU27:20.0)HTU23:17.5)HTU21:10.5,(ArmenianWestern:26.5,(SanskritVedic:31.0,(AncientIonicGreek:41.5,Irish:32.5)HTU18:31.0)HTU19:26.0)HTU30:21.0)HTU20:16.0)HTU29:29.0)HTU16[775.0];
(AlbanianTosk:24.5,(Danish:35.0,(((OldPrussian:34.0,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:13.5,(((Cornish:29.5,Manx:20.5)HTU17:25.0,(Yiddish:33.0,(EasternFrisianSaterland:34.0,KolschColognian:47.0)HTU26:14.5)HTU28:22.0)HTU25:17.5,(Assamese:42.0,Scots:40.0)HTU27:20.0)HTU23:17.5)HTU21:10.5,(ArmenianWestern:26.5,(SanskritVedic:31.0,(AncientIonicGreek:41.5,Irish:32.5)HTU18:31.0)HTU19:26.0)HTU30:21.0)HTU20:16.0)HTU29:29.0)HTU16[775.0];
(AlbanianTosk:24.5,(Danish:35.0,(((OldPrussian:34.0,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:13.5,(((Cornish:29.5,Manx:20.5)HTU17:25.0,(Yiddish:33.0,(EasternFrisianSaterland:34.0,KolschColognian:47.0)HTU26:14.5)HTU28:22.0)HTU25:17.5,(Assamese:42.0,Scots:40.0)HTU27:20.0)HTU23:17.5)HTU21:10.5,(ArmenianWestern:26.5,(SanskritVedic:31.0,(AncientIonicGreek:41.5,Irish:32.5)HTU18:31.0)HTU19:26.0)HTU30:21.0)HTU20:16.0)HTU29:29.0)HTU16[775.0];
(AlbanianTosk:25.0,(Danish:34.5,(((OldPrussian:33.5,(Faroese:25.0,Icelandic:59.0)HTU25:18.0)HTU23:10.5,(((Cornish:30.5,Manx:19.5)HTU18:24.0,(EasternFrisianSaterland:26.5,Yiddish:44.5)HTU28:18.5)HTU26:18.0,(Assamese:44.5,Scots:37.5)HTU27:21.0)HTU24:18.5)HTU22:6.5,(ArmenianWestern:31.0,(KolschColognian:48.5,(SanskritVedic:32.5,(AncientIonicGreek:40.5,Irish:33.5)HTU19:22.0)HTU20:22.5)HTU17:22.5)HTU30:18.0)HTU21:15.0)HTU29:28.0)HTU16[774.0];
(AlbanianTosk:25.0,(Danish:34.5,(((OldPrussian:33.5,(Faroese:25.0,Icelandic:59.0)HTU25:18.0)HTU23:10.5,(((Cornish:30.5,Manx:19.5)HTU18:24.0,(EasternFrisianSaterland:26.5,Yiddish:44.5)HTU28:18.5)HTU26:18.0,(Assamese:44.5,Scots:37.5)HTU27:21.0)HTU24:18.5)HTU22:6.5,(ArmenianWestern:31.0,(KolschColognian:48.5,(SanskritVedic:32.5,(AncientIonicGreek:40.5,Irish:33.5)HTU19:22.0)HTU20:22.5)HTU17:22.5)HTU30:18.0)HTU21:15.0)HTU29:28.0)HTU16[774.0];
(AlbanianTosk:24.0,(Danish:37.0,(((OldPrussian:36.0,(Faroese:24.5,Icelandic:59.5)HTU23:19.5)HTU21:13.0,(((Irish:26.0,(Cornish:26.5,Manx:23.5)HTU17:24.5)HTU25:18.5,(Yiddish:30.5,(EasternFrisianSaterland:33.5,KolschColognian:47.5)HTU26:16.0)HTU28:19.0)HTU24:21.0,(Assamese:41.5,Scots:40.5)HTU27:16.5)HTU22:18.5)HTU20:8.0,(ArmenianWestern:27.0,(AncientIonicGreek:48.5,SanskritVedic:25.5)HTU18:24.5)HTU30:20.0)HTU19:16.0)HTU29:27.0)HTU16[770.0];
(AlbanianTosk:26.0,(Danish:33.0,(((OldPrussian:34.5,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:11.0,(((Irish:25.5,(Cornish:25.5,Manx:24.5)HTU18:24.0)HTU26:16.5,(EasternFrisianSaterland:30.5,Yiddish:40.5)HTU28:12.5)HTU25:20.0,(Assamese:42.5,Scots:39.5)HTU27:18.0)HTU23:20.0)HTU21:8.0,(ArmenianWestern:32.0,(KolschColognian:46.0,(AncientIonicGreek:41.0,SanskritVedic:32.0)HTU19:24.0)HTU17:20.0)HTU30:18.0)HTU20:14.0)HTU29:29.0)HTU16[769.0];
(AlbanianTosk:26.0,(Danish:33.0,(((OldPrussian:34.5,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:11.0,(((Irish:25.5,(Cornish:25.5,Manx:24.5)HTU18:24.0)HTU26:16.5,(EasternFrisianSaterland:30.5,Yiddish:40.5)HTU28:12.5)HTU25:20.0,(Assamese:42.5,Scots:39.5)HTU27:18.0)HTU23:20.0)HTU21:8.0,(ArmenianWestern:32.0,(KolschColognian:46.0,(AncientIonicGreek:41.0,SanskritVedic:32.0)HTU19:24.0)HTU17:20.0)HTU30:18.0)HTU20:14.0)HTU29:29.0)HTU16[769.0];
(AlbanianTosk:26.0,(Danish:33.0,(((OldPrussian:34.5,(Faroese:25.5,Icelandic:58.5)HTU24:18.5)HTU22:11.0,(((Irish:25.5,(Cornish:25.5,Manx:24.5)HTU18:24.0)HTU26:16.5,(EasternFrisianSaterland:30.5,Yiddish:40.5)HTU28:12.5)HTU25:20.0,(Assamese:42.5,Scots:39.5)HTU27:18.0)HTU23:20.0)HTU21:8.0,(ArmenianWestern:32.0,(KolschColognian:46.0,(AncientIonicGreek:41.0,SanskritVedic:32.0)HTU19:24.0)HTU17:20.0)HTU30:18.0)HTU20:14.0)HTU29:29.0)HTU16[769.0];
