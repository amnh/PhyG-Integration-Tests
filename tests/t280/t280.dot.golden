digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    16 -> 0 [label=30.5];
    16 -> 24 [label=31.5];
    17 -> 4 [label=27.5];
    17 -> 11 [label=22.5];
    18 -> 12 [label=43.5];
    18 -> 21 [label=12.0];
    19 -> 2 [label=33.0];
    19 -> 34 [label=296.0];
    21 -> 19 [label=22.5];
    21 -> 33 [label=296.5];
    22 -> 6 [label=31.0];
    22 -> 23 [label=18.5];
    23 -> 9 [label=27.0];
    23 -> 15 [label=40.0];
    24 -> 18 [label=10.5];
    24 -> 22 [label=22.0];
    24 -> 27 [label=7.5];
    25 -> 17 [label=24.0];
    25 -> 31 [label=19.5];
    26 -> 3 [label=39.5];
    26 -> 14 [label=42.5];
    27 -> 28 [label=16.0];
    27 -> 33 [label=293.5];
    28 -> 25 [label=19.5];
    28 -> 34 [label=292.5];
    29 -> 7 [label=27.0];
    29 -> 8 [label=57.0];
    30 -> 5 [label=38.0];
    30 -> 26 [label=21.5];
    31 -> 13 [label=35.0];
    31 -> 32 [label=23.0];
    32 -> 29 [label=19.0];
    32 -> 30 [label=18.0];
    33 -> 1 [label=291.0];
    34 -> 10 [label=290.0];
}
//897.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=28.5];
    16 -> 24 [label=31.0];
    17 -> 4 [label=28.0];
    17 -> 11 [label=22.0];
    18 -> 12 [label=37.5];
    18 -> 21 [label=18.0];
    19 -> 2 [label=32.0];
    19 -> 10 [label=49.5];
    20 -> 18 [label=18.5];
    20 -> 22 [label=19.0];
    21 -> 1 [label=36.0];
    21 -> 19 [label=23.0];
    22 -> 6 [label=28.5];
    22 -> 23 [label=16.5];
    23 -> 9 [label=29.0];
    23 -> 15 [label=38.0];
    24 -> 20 [label=8.5];
    24 -> 25 [label=12.5];
    25 -> 17 [label=22.0];
    25 -> 29 [label=17.0];
    26 -> 3 [label=40.5];
    26 -> 14 [label=41.5];
    27 -> 7 [label=26.5];
    27 -> 8 [label=57.5];
    28 -> 5 [label=38.5];
    28 -> 26 [label=20.0];
    29 -> 13 [label=39.5];
    29 -> 30 [label=19.5];
    30 -> 27 [label=18.5];
    30 -> 28 [label=17.5];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=27.0];
    16 -> 23 [label=29.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 12 [label=35.5];
    18 -> 20 [label=18.5];
    20 -> 1 [label=41.0];
    20 -> 2 [label=38.0];
    21 -> 6 [label=24.0];
    21 -> 22 [label=20.0];
    22 -> 9 [label=27.0];
    22 -> 15 [label=40.0];
    23 -> 18 [label=19.5];
    23 -> 21 [label=18.5];
    23 -> 26 [label=10.5];
    24 -> 17 [label=24.0];
    24 -> 29 [label=17.5];
    25 -> 3 [label=39.5];
    25 -> 14 [label=42.5];
    26 -> 10 [label=52.0];
    26 -> 24 [label=14.5];
    27 -> 7 [label=27.5];
    27 -> 8 [label=56.5];
    28 -> 5 [label=39.0];
    28 -> 25 [label=21.0];
    29 -> 13 [label=37.0];
    29 -> 30 [label=21.0];
    30 -> 27 [label=19.5];
    30 -> 28 [label=19.0];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=27.0];
    16 -> 23 [label=29.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 12 [label=35.5];
    18 -> 20 [label=18.5];
    20 -> 1 [label=41.0];
    20 -> 2 [label=38.0];
    21 -> 6 [label=24.0];
    21 -> 22 [label=20.0];
    22 -> 9 [label=27.0];
    22 -> 15 [label=40.0];
    23 -> 18 [label=19.5];
    23 -> 21 [label=18.5];
    23 -> 26 [label=10.5];
    24 -> 17 [label=24.0];
    24 -> 29 [label=17.5];
    25 -> 3 [label=39.5];
    25 -> 14 [label=42.5];
    26 -> 10 [label=52.0];
    26 -> 24 [label=14.5];
    27 -> 7 [label=27.5];
    27 -> 8 [label=56.5];
    28 -> 5 [label=39.0];
    28 -> 25 [label=21.0];
    29 -> 13 [label=37.0];
    29 -> 30 [label=21.0];
    30 -> 27 [label=19.5];
    30 -> 28 [label=19.0];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=27.0];
    16 -> 23 [label=29.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 12 [label=35.5];
    18 -> 20 [label=18.5];
    20 -> 1 [label=41.0];
    20 -> 2 [label=38.0];
    21 -> 6 [label=24.0];
    21 -> 22 [label=20.0];
    22 -> 9 [label=27.0];
    22 -> 15 [label=40.0];
    23 -> 18 [label=19.5];
    23 -> 21 [label=18.5];
    23 -> 26 [label=10.5];
    24 -> 17 [label=24.0];
    24 -> 29 [label=17.5];
    25 -> 3 [label=39.5];
    25 -> 14 [label=42.5];
    26 -> 10 [label=52.0];
    26 -> 24 [label=14.5];
    27 -> 7 [label=27.5];
    27 -> 8 [label=56.5];
    28 -> 5 [label=39.0];
    28 -> 25 [label=21.0];
    29 -> 13 [label=37.0];
    29 -> 30 [label=21.0];
    30 -> 27 [label=19.5];
    30 -> 28 [label=19.0];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=28.5];
    16 -> 24 [label=31.0];
    17 -> 4 [label=28.0];
    17 -> 11 [label=22.0];
    18 -> 12 [label=37.5];
    18 -> 21 [label=18.0];
    19 -> 2 [label=32.0];
    19 -> 10 [label=49.5];
    20 -> 18 [label=18.5];
    20 -> 22 [label=19.0];
    21 -> 1 [label=36.0];
    21 -> 19 [label=23.0];
    22 -> 6 [label=28.5];
    22 -> 23 [label=16.5];
    23 -> 9 [label=29.0];
    23 -> 15 [label=38.0];
    24 -> 20 [label=8.5];
    24 -> 25 [label=12.5];
    25 -> 17 [label=22.0];
    25 -> 29 [label=17.0];
    26 -> 3 [label=40.5];
    26 -> 14 [label=41.5];
    27 -> 7 [label=26.5];
    27 -> 8 [label=57.5];
    28 -> 5 [label=38.5];
    28 -> 26 [label=20.0];
    29 -> 13 [label=39.5];
    29 -> 30 [label=19.5];
    30 -> 27 [label=18.5];
    30 -> 28 [label=17.5];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=27.0];
    16 -> 23 [label=29.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 12 [label=35.5];
    18 -> 20 [label=18.5];
    20 -> 1 [label=41.0];
    20 -> 2 [label=38.0];
    21 -> 6 [label=24.0];
    21 -> 22 [label=20.0];
    22 -> 9 [label=27.0];
    22 -> 15 [label=40.0];
    23 -> 18 [label=19.5];
    23 -> 21 [label=18.5];
    23 -> 26 [label=10.5];
    24 -> 17 [label=24.0];
    24 -> 29 [label=17.5];
    25 -> 3 [label=39.5];
    25 -> 14 [label=42.5];
    26 -> 10 [label=52.0];
    26 -> 24 [label=14.5];
    27 -> 7 [label=27.5];
    27 -> 8 [label=56.5];
    28 -> 5 [label=39.0];
    28 -> 25 [label=21.0];
    29 -> 13 [label=37.0];
    29 -> 30 [label=21.0];
    30 -> 27 [label=19.5];
    30 -> 28 [label=19.0];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=28.5];
    16 -> 24 [label=31.0];
    17 -> 4 [label=28.0];
    17 -> 11 [label=22.0];
    18 -> 12 [label=37.5];
    18 -> 21 [label=18.0];
    19 -> 2 [label=32.0];
    19 -> 10 [label=49.5];
    20 -> 18 [label=18.5];
    20 -> 22 [label=19.0];
    21 -> 1 [label=36.0];
    21 -> 19 [label=23.0];
    22 -> 6 [label=28.5];
    22 -> 23 [label=16.5];
    23 -> 9 [label=29.0];
    23 -> 15 [label=38.0];
    24 -> 20 [label=8.5];
    24 -> 25 [label=12.5];
    25 -> 17 [label=22.0];
    25 -> 29 [label=17.0];
    26 -> 3 [label=40.5];
    26 -> 14 [label=41.5];
    27 -> 7 [label=26.5];
    27 -> 8 [label=57.5];
    28 -> 5 [label=38.5];
    28 -> 26 [label=20.0];
    29 -> 13 [label=39.5];
    29 -> 30 [label=19.5];
    30 -> 27 [label=18.5];
    30 -> 28 [label=17.5];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=28.5];
    16 -> 24 [label=31.0];
    17 -> 4 [label=28.0];
    17 -> 11 [label=22.0];
    18 -> 12 [label=37.5];
    18 -> 21 [label=18.0];
    19 -> 2 [label=32.0];
    19 -> 10 [label=49.5];
    20 -> 18 [label=18.5];
    20 -> 22 [label=19.0];
    21 -> 1 [label=36.0];
    21 -> 19 [label=23.0];
    22 -> 6 [label=28.5];
    22 -> 23 [label=16.5];
    23 -> 9 [label=29.0];
    23 -> 15 [label=38.0];
    24 -> 20 [label=8.5];
    24 -> 25 [label=12.5];
    25 -> 17 [label=22.0];
    25 -> 29 [label=17.0];
    26 -> 3 [label=40.5];
    26 -> 14 [label=41.5];
    27 -> 7 [label=26.5];
    27 -> 8 [label=57.5];
    28 -> 5 [label=38.5];
    28 -> 26 [label=20.0];
    29 -> 13 [label=39.5];
    29 -> 30 [label=19.5];
    30 -> 27 [label=18.5];
    30 -> 28 [label=17.5];
}
//784.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=26.0];
    16 -> 22 [label=27.5];
    17 -> 4 [label=28.0];
    17 -> 11 [label=22.0];
    18 -> 2 [label=32.5];
    18 -> 12 [label=40.5];
    19 -> 18 [label=13.0];
    19 -> 20 [label=19.0];
    20 -> 6 [label=27.5];
    20 -> 21 [label=19.0];
    21 -> 9 [label=27.5];
    21 -> 15 [label=39.5];
    22 -> 19 [label=5.0];
    22 -> 25 [label=15.0];
    23 -> 17 [label=23.0];
    23 -> 29 [label=16.5];
    24 -> 3 [label=39.5];
    24 -> 14 [label=42.5];
    25 -> 1 [label=44.5];
    25 -> 26 [label=14.0];
    26 -> 10 [label=50.0];
    26 -> 23 [label=17.5];
    27 -> 7 [label=27.5];
    27 -> 8 [label=56.5];
    28 -> 5 [label=38.5];
    28 -> 24 [label=21.0];
    29 -> 13 [label=34.5];
    29 -> 30 [label=23.5];
    30 -> 27 [label=19.5];
    30 -> 28 [label=18.5];
}
//782.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=25.5];
    16 -> 23 [label=27.5];
    17 -> 4 [label=26.0];
    17 -> 11 [label=24.0];
    18 -> 12 [label=37.5];
    18 -> 19 [label=20.5];
    19 -> 2 [label=36.5];
    19 -> 10 [label=44.5];
    20 -> 18 [label=18.5];
    20 -> 21 [label=20.0];
    21 -> 6 [label=27.0];
    21 -> 22 [label=16.0];
    22 -> 9 [label=27.0];
    22 -> 15 [label=40.5];
    23 -> 20 [label=11.5];
    23 -> 26 [label=16.0];
    24 -> 17 [label=20.0];
    24 -> 29 [label=18.0];
    25 -> 3 [label=40.5];
    25 -> 14 [label=41.5];
    26 -> 1 [label=44.5];
    26 -> 24 [label=11.5];
    27 -> 7 [label=27.0];
    27 -> 8 [label=57.0];
    28 -> 5 [label=38.5];
    28 -> 25 [label=21.0];
    29 -> 13 [label=38.5];
    29 -> 30 [label=22.5];
    30 -> 27 [label=20.0];
    30 -> 28 [label=19.0];
}
//781.0
